web-client/src/views/FileHandlingHelpers/pdfValidation.ts

```diff
@@ -12,6 +12,13 @@ export const PDF_CORRUPTED_ERROR_MESSAGE =
 const GENERIC_FILE_ERROR_MESSAGE =
   'There is a problem uploading the file. Try again later.';
 
+export const validatePdfHeader = (pdfData: Uint8Array): boolean => {
+  const stringDecoder = new TextDecoder('utf8');
+  const pdfHeaderBytes = pdfData.slice(0, 5);
+  const pdfHeaderString = stringDecoder.decode(pdfHeaderBytes);
+  return pdfHeaderString === '%PDF-';
+};
```

Comment:
We do this on the back end [here](https://github.com/ustaxcourt/ef-cms/blob/263df266f4d323a93eca085d9d03a85de1177272/web-api/src/business/useCases/pdf/validatePdfInteractor.ts#L33-L52). I was hoping to have a shared function so that both places used the same logic, but there were enough differences (mainly logging and using a different decoder class) that doing so proved more complicated rather than less.